itest/rust/build.rs

```diff
@@ -269,6 +269,10 @@ fn main() {
     rustfmt_if_needed(vec![rust_file]);
 
     godot_bindings::emit_godot_version_cfg();
+
+    if godot_codegen::IS_CODEGEN_FULL {
+        println!("cargo::rustc-cfg=feature=\"codegen-full\"");
+    }
```

Comment:
Could you add some description what this solves?

(if it helps, some inspiration is [here](https://github.com/godot-rust/gdext/pull/1101/files/b66f56d5e62abfefe969178e136557266e4189b4..614b71df2838698b4e662161435f3d04ca1428b5); feel free to use your own words though)

Reply:
Added a comment. I hope it clarifies the problem clear.